.github/workflows/osv-scanner-scan.yml

```
@@ -18,4 +18,29 @@ jobs:
       contents: read # to fetch code (actions/checkout)
       security-events: write # for uploading SARIF files
     if: ${{ github.repository == 'metal3-io/ip-address-manager' && github.event_name == 'schedule' }}
```

```suggestion
    if: ${{ github.repository == 'metal3-io/ip-address-manager' }}
```
We only trigger from schedule, we can remove extra check.